client/reader/stream/index.jsx

```diff
@@ -665,6 +667,9 @@ class ReaderStream extends Component {
 				body = (
 					<>
 						{ streamHeader?.() }
+						<div style={ { margin: '32px 16px 0' } }>
```

Comment:
Creating space on mobile; didn't want to add another stylesheet since this is temporary.